src/index.ts

```diff
@@ -0,0 +1,79 @@
+import * as Animators from "./animators";
```

Comment:
The whitespace reflects the mapping between file hierarchy to namespace hierarchy, especially because the exports vary later on.

Oh I just realized I forgot to be consistent with these first three... will change